### ISSUE TITLE
Render XML without escaping

### ISSUE THREAD

**Issue by ear7h** [dhall-lang/dhall-lang#1224]
I would like to do something like:

```dhall
let blogPost =
  [ XML/element = {name = "h1", attributes = [] : Attrs, content = [ XML/text "Blog Post!" ]
  ] # XML/rawText (./blog-post.html as Text)
```

I've been looking at the XML package and it looks like this isn't possible. I think it would take changing the occurrences of
```dhall
{ text : Text → XML
, element :
	{ attributes : List { mapKey : Text, mapValue : Text }
	, content : List XML
	, name : Text
	} → XML
}
```
to
```dhall
{ text : Text → XML
, rawText : Text → XML
, element :
	{ attributes : List { mapKey : Text, mapValue : Text }
	, content : List XML
	, name : Text
	} → XML
}
```

**Comment by keks** [comment]
I believe this is fixed.

**Comment by Gabriella439** [comment]
Yeah, you're right.  This is fixed by #1225
